### DISCUSSION TITLE
Where to add images?

### DISCUSSION THREAD

**Discussion by Axehaft** [TandoorRecipes/recipes#2646] [General]
I would like to add images of nutrition information.  Where can I do that?  

Maybe a better question = is there a way to add nutrition info?

**Comment by smilerz** [comment]
Nutrition info is available under "additional options"

**Reply by Axehaft** [reply]
I don't see that?

![image](https://github.com/TandoorRecipes/recipes/assets/46401365/6ed9c135-8548-433f-8f33-e27adf4523c3)
